packages/transport/src/core/backbone/RESTClient.ts

```diff
@@ -81,17 +82,24 @@ export class RESTClient {
             defaults.headers = _.defaultsDeep({}, defaults.headers, this.config.platformAdditionalHeaders);
         }
 
+        if (config.httpsAgent) {
+            defaults.httpsAgent = config.httpsAgent;
+        } else {
+            try {
+                // eslint-disable-next-line @typescript-eslint/no-require-imports
+                const httpsAgent = require("https")?.Agent;
```

Comment:
Why is this required here when it is imported at the start of the file ?

Reply:
it's not available in browsers